Update storage-use-azcopy-files.md

SurbhiSingh1712 opened the pull request:
Updated Note section with below details:

For smaller blobs (<256MB IIRC), the service computes md5 hash code automatically.

SurbhiSingh1712 committed:
Update storage-use-azcopy-files.md

Updated Note section with below details:

For smaller blobs (<256MB IIRC), the service computes md5 hash code automatically.

normesta commented on articles/storage/common/storage-use-azcopy-files.md:
```diff
@@ -57,7 +57,7 @@ This section contains the following examples:
 
 > [!NOTE]
-> AzCopy doesn't automatically calculate and store the file's md5 hash code. If you want AzCopy to do that, then append the `--put-md5` flag to each copy command. That way, when the file is downloaded, AzCopy calculates an MD5 hash for downloaded data and verifies that the MD5 hash stored in the file's `Content-md5` property matches the calculated hash.
+> AzCopy doesn't automatically calculate and store the file's md5 hash code .If you want AzCopy to do that, then append the `--put-md5` flag to each copy command. That way, when the file is downloaded, AzCopy calculates an MD5 hash for downloaded data and verifies that the MD5 hash stored in the file's `Content-md5` property matches the calculated hash.For smaller blobs (<256MB IIRC), the service ...
```
Hi @SurbhiSingh1712 - thanks for the contribution! I have a question about this proposed change. The line you're adding at the end here appears to contradict the rest of this note. The note states that the MD5 hash code is no automatically calculated. Your addition at the end states that it is (albeit in small files). Am I reading this proposed change correctly? Also, on a smaller note, it appears that you accidentally added a space at the end of the first sentence of this note.

SurbhiSingh1712 committed:
Update storage-use-azcopy-files.md

Corrected the sentence in Note
AzCopy doesn't automatically calculate and store the file's md5 hash code for blobs (>256 MB)

SurbhiSingh1712 committed:
Update storage-use-azcopy-files.md

Updated Note :

AzCopy doesn't automatically calculates and stores the file's md5 hash code for a file greater than 256 MB

Court72 commented on articles/storage/common/storage-use-azcopy-files.md:
```diff
@@ -57,7 +57,7 @@ This section contains the following examples:
 
 > [!NOTE]
-> AzCopy doesn't automatically calculate and store the file's md5 hash code. If you want AzCopy to do that, then append the `--put-md5` flag to each copy command. That way, when the file is downloaded, AzCopy calculates an MD5 hash for downloaded data and verifies that the MD5 hash stored in the file's `Content-md5` property matches the calculated hash.
+> AzCopy doesn't automatically calculates and stores the file's md5 hash code for a file greater than 256 MB.If you want AzCopy to do that, then append the `--put-md5` flag to each copy command. That way, when the file is downloaded, AzCopy calculates an MD5 hash for downloaded data and verifies that the MD5 hash stored in the file's `Content-md5` property matches the calculated hash.
```
```suggestion
> AzCopy doesn't automatically calculate and store the file's md5 hash code for a file greater than 256 MB.If you want AzCopy to do that, then append the `--put-md5` flag to each copy command. That way, when the file is downloaded, AzCopy calculates an MD5 hash for downloaded data and verifies that the MD5 hash stored in the file's `Content-md5` property matches the calculated hash.
```

Court72 committed:
Update articles/storage/common/storage-use-azcopy-files.md
